doc/man3/SHA256_Init.pod

```diff
@@ -49,7 +49,12 @@ L<openssl_user_macros(7)>:
 
 All of the functions described on this page are deprecated.
 Applications should instead use L<EVP_DigestInit_ex(3)>, L<EVP_DigestUpdate(3)>
-and L<EVP_DigestFinal_ex(3)>.
+and L<EVP_DigestFinal_ex(3)>, or L<EVP_Q_digest(3)>.
+SHA1(), SHA224(), SHA256(), SHA384(), and SHA256()
+can continue to be used by defining B<OPENSSL_COMPENSATE_DEPRECATION>.
```

Comment:
This comment is still here?

Reply:
Ops, yes. Re-worked this in the same way as above to:
```
and L<EVP_DigestFinal_ex(3)>, or the quick one-shot function L<EVP_Q_digest(3)>.
SHA1(), SHA224(), SHA256(), SHA384(), and SHA256()
can continue to be used. They can also be replaced by, e.g.,

    (EVP_Q_digest(d, n, md, NULL, NULL, "SHA256", NULL) ? md : NULL)
```
Doing so, I noticed that the synopsis of the following functions had to be moved up because they are no more deprecated. I've also updated their arg names and types:
```
 unsigned char *SHA1(const void *data, size_t count, unsigned char *md_buf);
 unsigned char *SHA224(const void *data, size_t count, unsigned char *md_buf);
 unsigned char *SHA256(const void *data, size_t count, unsigned char *md_buf);
 unsigned char *SHA384(const void *data, size_t count, unsigned char *md_buf);
 unsigned char *SHA512(const void *data, size_t count, unsigned char *md_buf);
```